src/build/strings.ts

```diff
@@ -27,6 +27,9 @@ const strings = {
   'extensionHost.snippet.launch.description': 'Launch a VS Code extension in debug mode',
   'extensionHost.snippet.launch.label': 'VS Code Extension Development',
 
+  'edge.useWebView.description':
+    "(Edge only) When 'true', the debugger will treat the runtime executable as a host application that contains a WebView allowing you to debug the WebView script content. When set to 'advanced', the debugger will wait to connect to a WebView matching the urlFilter.",
```

Comment:
Should this say "Edge (Chromium)" only?

Reply:
This prefix should be removed as part of #266

Reply:
Done.